app/javascript/components/pages/LastCallEdit.js

```diff
@@ -1,169 +1,189 @@
-import React, { useEffect } from 'react'
-import { useState, useRef } from 'react'
-import { Form, Button, Container, Col, Row } from 'react-bootstrap'
-import { Navigate, useParams } from 'react-router-dom'
+import React, { useEffect } from "react";
+import { useState, useRef } from "react";
+import { Form, Button, Container, Col, Row } from "react-bootstrap";
+import { Navigate, useParams } from "react-router-dom";
 
 export default function LastCallEdit(props) {
-  let { id } = useParams()
-  let { editCocktail, current_user, logged_in } = props
-  
+  let { id } = useParams();
+  let { editCocktail, current_user, logged_in } = props;
 
-  const [submit, setSubmit] = useState(false)
+  const [submit, setSubmit] = useState(false);
   const [cocktailEdit, setCocktailEdit] = useState({
     name: "",
     image: "",
     ingredients: [""],
     directions: [""],
-    user_id: current_user.id
-  })
+    user_id: current_user.id,
+  });
 
-  const [ingredient, setIngredient] = useState("")
-  const [list, setList] = useState([])
+  const [ingredient, setIngredient] = useState("");
+  const [list, setList] = useState([]);
 
-  const [direction, setDirection] = useState("")
-  const [listDir, setListDir] = useState([])
+  const [direction, setDirection] = useState("");
+  const [listDir, setListDir] = useState([]);
 
-  const ingRef = useRef(null)
-  const dirRef = useRef(null)
+  const ingRef = useRef(null);
+  const dirRef = useRef(null);
 
-  let [cocktail, setCocktail] = useState([])
-  
+  let [cocktail, setCocktail] = useState([]);
 
-  console.log("show:", logged_in)
-  console.log("data", cocktail.user_id)
-  function showCocktails (id) {
+  console.log("show:", logged_in);
+  console.log("data", cocktail.user_id);
+  function showCocktails(id) {
     fetch(`/cocktails/${id}`)
-    .then(request => request.json())
-    .then(payload => setCocktail(payload))
-    .catch(err => console.log(err))
+      .then((request) => request.json())
+      .then((payload) => setCocktail(payload))
+      .catch((err) => console.log(err));
   }
 
   function handleSubmit(e) {
-    e.preventDefault()
-    editCocktail(cocktailEdit, id)
-    setSubmit(true)
+    e.preventDefault();
+    editCocktail(cocktailEdit, id);
+    setSubmit(true);
   }
 
   function handleCocktailArray(ing, e) {
-    e.preventDefault()
+    e.preventDefault();
```

Comment:
e.preventDefault is only needed for forms.
for regular buttons you don't need it.

event handler (e) isin't being used anywhere in this function; remove it.